src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/BeneficialOwnerAdditionService.java

```diff
@@ -0,0 +1,149 @@
+package uk.gov.companieshouse.overseasentitiesapi.service;
```

Comment:
We're missing logs on all our new services. That's a debate amongst the team on whether we do them at the end or as we go. I'd say as-we-go would be handier, but we'd need a ticket to add those that aren't logged already.